api/queries_issue.go

```diff
@@ -122,6 +122,40 @@ func IssueCreate(client *Client, repo *Repository, params map[string]interface{}
 	return &result.CreateIssue.Issue, nil
 }
 
+func CommentCreate(client *Client, repoHost string, params map[string]string) (string, error) {
+	query := `
+	mutation CommentCreate($input: AddCommentInput!) {
+		addComment(input: $input) {
+			commentEdge {
+				node {
+					url
+				}
+			}
+		}
+	}`
+
+	variables := map[string]interface{}{
+		"input": params,
+	}
+
+	result := struct {
+		AddComment struct {
+			CommentEdge struct {
+				Node struct {
+					URL string
+				}
+			}
+		}
+	}{}
+
+	err := client.GraphQL(repoHost, query, variables, &result)
```

Comment:
Per our Slack convo, you could use the `github.com/shurcooL/graphql` library to simplify the GraphQL interaction here 👍